### DISCUSSION TITLE
[2024-09-02] GUI Team Demo Day

### DISCUSSION THREAD

**Discussion by vitvakatu** [enso-org/enso#10951] [Weekly Demo Day]
- https://github.com/enso-org/enso/pull/10920 – issue with viewport jumping was caused by a race condition when creating new nodes.
- https://github.com/enso-org/enso/pull/10900 – e2e tests are flaky, but the panel works.
- https://github.com/enso-org/enso/pull/10937 – when changing the direction of CB list we forgot to check some edge-cases
- https://github.com/enso-org/enso/pull/10950 – another sort-of-race-condition in deleting nodes.

https://github.com/user-attachments/assets/12f3d155-b4ea-4c78-a8a5-fb771c80f3f2

**Comment by kazcw** [comment]
- #10876

https://github.com/user-attachments/assets/e88801c2-8c56-494b-8500-95fbd76c4322

- #10763

- #10691

https://github.com/user-attachments/assets/7daa342d-a7f1-4f63-b934-1519724a59a5

- #10543
- #10312 -- Chrome fixed this one

**Comment by Frizi** [comment]
- https://github.com/enso-org/enso/issues/10508 - configuring bazel build to produce GUI artifacts, including required WASM files - Still in progress

**Comment by JaroslavTulach** [comment]
### Engine interesting work

- #10751 vs. JVM version blocked by a GraalVM bug - [backport requested](https://github.com/oracle/graal/pull/9516#issuecomment-2311796280)
- #10872 - when Ydoc server retries to connect to language server

![still 10%](https://github.com/user-attachments/assets/d62eb95f-a1a8-42b2-be31-482f7cdd0607)

- Benchmark and speed processing of polyglot java imports up #10899
- #10837
- #10906
- #10842
